package.json

```diff
@@ -31,18 +31,18 @@
     },
     "homepage": "https://github.com/thegc/html-webpack-inline-svg-plugin",
```

Comment:
On line 3 of `package.json` the version of the package has to be bumped up, perhaps to `1.3.1`?

Reply:
Definitely the version should change, but I didn't update it because I didn't know if it would be `1.3.1` or `1.4.0` or even `1.3.2` if another PR is merged before this one.

Should I set it to `1.3.1`?

Reply:
I set it to `1.3.1`